### PR TITLE
Addition of Roadmap

### PR DESCRIPTION
**Pull request by stuti-ui**

### CODE REVIEW

**Comment by stuti-ui** [comment]
@RitikPatni kindly look #107

**Review by RitikPatni — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -57,6 +58,35 @@ This is an open-source repository for all who want to learn front-end Developmen
   - [YouTube Videos](#youtube-videos)
   - [Testing](#testing)
 
+## Roadmap
```
[the remark this answers is not in the document]
Comment (stuti-ui) [reply README.md]: Changes done. Kindly review it once again.

[the remark this answers is not in the document]
Comment (RitikPatni) [reply README.md]: This looks much clear now. Thanks for the contribution.

**Comment by stuti-ui** [comment]
Thank you.

**Pull request merged**